lib/irc/IrcServer.js

```diff
@@ -526,7 +530,8 @@ IrcServer.DEFAULT_CONFIG = {
     mappings: {},
     matrixClients: {
         userTemplate: "@$SERVER_$NICK",
-        displayName: "$NICK (IRC)"
+        displayName: "$NICK (IRC)",
+        joinAttempts: -1,
```

Comment:
No , at the end here.

Reply:
https://github.com/matrix-org/matrix-appservice-irc/pull/622#discussion_r204739927 :)